### DISCUSSION TITLE
Html

### DISCUSSION THREAD

**Discussion by 123Will** [balta-io/2801#100] [Perguntas e Respostas]
![image](https://user-images.githubusercontent.com/97917861/186043318-245a2bc6-72f0-4782-827f-87147b83755f.png)
![image](https://user-images.githubusercontent.com/97917861/186043335-75350a6f-49f7-4610-9d15-ed9fa3d4bd9d.png)

Boa noite, quando eu coloco o menu.Show, ele da esse erro, já refiz o código mas não resolve, pode me ajudar por favo, obrigado!!!

**Comment by cmsouzaac** [comment]
@123Will como vc deixa seu terminal na parte de baixo do codigo? quando veio essa ultima atualização do vs  so está na lateral e nao consigo colocar em baixo.

![image](https://user-images.githubusercontent.com/48097255/186210310-317908f5-1e04-4e1e-bc2e-f038e63541c4.png)

**Comment by andrebaltieri** [comment]
@123Will sua classe `Show` está dentro da classe `Menu`, elas precisam estar no mesmo nível. `Menu.Show` chama uma função filha.

@cmsoouza clica com o botão direito sobre o **Terminal** e seleciona **Move To Bottom**

**Reply by 123Will** [reply]
@andrebaltieri  Boa noite, não entendi sobre a classe, eu ja refiz do jeito que esta a aula 5x, mas mesmo assim da erro ao criar o Menu.cs, o mesmo  fica vermelho, sendo que ja voltei todo o video fiz passo a passo e sempre da esse erro, não compreendi como devo resolver.

**Reply by 123Will** [reply]
@andrebaltieri Não consegui ver nada de diferente entre sua aula e meu código, ja revirei isso kkkk

**Comment by lasawally** [comment]
![image](https://user-images.githubusercontent.com/111128419/187483178-67911917-a200-44c7-8ea8-68675ec5d158.png)
![image](https://user-images.githubusercontent.com/111128419/187483246-06e0278c-f934-4c38-880c-698ea5ea2333.png)
![image](https://user-images.githubusercontent.com/111128419/187483600-2cc43763-3df2-43aa-a605-299ccf283c55.png)
![image](https://user-images.githubusercontent.com/111128419/187483646-45dd627c-e6cc-4f0b-aa42-ed7e139d8cae.png)


As classes Show e Viewer não são reconhecidas e não entendo o porquê disso.

**Reply by 123Will** [reply]
Na minha é assim também, e ta exactamente
 igual a aula

**Reply by lasawally** [reply]
siim, também não consigo entender oq houve. fiquei pensando se pode ser algo sobre a atualização do .NET

**Reply by 123Will** [reply]
Até agora não resolvi o problema .

**Reply by andrebaltieri** [reply]
Acredito que faltou o `static` na classe... `public static class Viewer...`
